ga4gh/datamodel/reads.py

```diff
@@ -182,19 +252,15 @@ def toProtocolElement(self):
         readGroup.referenceSetId = None
         readGroup.sampleId = None
         stats = protocol.ReadStats()
-        stats.alignedReadCount = self.getNumAlignedReads()
-        stats.unalignedReadCount = self.getNumUnalignedReads()
+        stats.alignedReadCount = \
+            self._parentContainer.getNumAlignedReads()
```

Comment:
True, but that requires iterating through all the reads in the read group.  What should we do here?  We could put some dummy value here.  Maybe we could even change the schema to remove readGroup.stats ...